app/views/rooms/index.html.erb

```diff
@@ -0,0 +1,30 @@
+<%= provide(:title, "Rooms") %>
+<div class="row mx-4">
+    <div class="col-2">
+        <%= render 'form' %>
+    </div>
+    <div class="col-10">
+        <div class="card container mt-4">
+            <div class="card-header">
+                <h3>Your Listings<h3>
+            </div>   
+                <% @rooms.each do |room| %> 
+                <div class="card-body row">            
+                    <%= image_tag room.images.first.image, style: 'width: 100px; height: 100px;' %>
+                    <div class="ml-3 mt-3">
+                        <h3><%= room.listing_name %></h3>
+                        <div class="text-align-bottom text-danger">
+                            <%= link_to raw("<i class='fa fa-trash text-danger'></i>"), room_path(room), method: :delete, data: { confirm: "You sure?" } %> <%= "( Delete )" %>
```

Comment:
Please fix a minor bug where if i click on the word Delete, nothing happens..

_HINT:_
just move the word delete after the `</i>` tag